packages/template-chakra-storefront/src/hooks/use-wish-list.js

```diff
@@ -5,17 +5,43 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import {useEffect} from 'react'
-import {useCustomerProductLists, useShopperCustomersMutation} from '@salesforce/commerce-sdk-react'
-import {useCurrentCustomer} from './use-current-customer'
+import React, {useEffect, useCallback} from 'react'
```

Comment:
Nice! Should we make this pattern (Combine all the functionality of an object into one hook) our standard/recommended approach?

Reply:
we should document somewhere so that all the teams are aware. I believe @vmarta is creating one